docs/metadata.md

```diff
@@ -77,3 +77,24 @@ Cozy-Client will:
 
 ⚠️ These values will override any values that your document may already have.
 
+### Specific case of io.cozy.files
+
+`io.cozy.files` attribut cozyMetadata is protected by cozy-stack and can't be saved directly.
```

Comment:
Reformulation suggestion: 
The `io.cozy.files` doctype is protected by cozy-stack, that restricts the attributes that can be saved by the client, including the `cozyMetadata`.

Reply:
Done